bread/bread.py

```diff
@@ -359,6 +359,39 @@ def __init__(self,
                                       or setting('DEFAULT_TEMPLATE_NAME_PATTERN', None))
         self.url_namespace = url_namespace
 
+        if self.columns:
+            self.validate_columns()
+
+    def validate_columns(self):
+        for title, column in self.columns:
+            self.validate_column(column)
+
+    def validate_column(self, column, model=None):
+        """
+        A column specifier can be a simple name, which should refer to a field
+        on the model, or can have multiple parts separated by '__', in which case
+        the first part must be the name of a field that is a ForeignKey or other kind
+        of field that refers to another model. Then the next part of the column
+        specifier must be a field name on that model, and so on.
+
+        Raises a FieldDoesNotExist or AssertionError if there's anything noticed
+        that is not correct.
+        """
```

Comment:
It's reasonable that this method always returns `None`, but it's not obvious without reading the code. It seems worth mentioning in the docstring.